sdcm/tester.py

```diff
@@ -891,8 +893,42 @@ def verify_gemini_results(self, queue):
                          'errors': stats['errors']})
         return stats
 
+    def run_fullscan(self, ks_cf, loader_node, db_node, page_size=100000):
+        """Run cql select count(*) request
+
+        if ks_cf is not random, use value from config
+        if ks_cf is random, choose random from not system
+
+        Arguments:
+            loader_node {BaseNode} -- loader cluster node
+            db_node {BaseNode} -- db cluster node
+
+        Returns:
+            object -- object with result of remoter.run command
+        """
+        ks_cf_list = self.loaders.get_non_system_ks_cf_list(loader_node, db_node)
+        if ks_cf not in ks_cf_list:
+            ks_cf = 'random'
+
+        if 'random' in ks_cf.lower():
+            ks_cf = random.choice(ks_cf_list)
+
+        FullScanEvent(type='start', db_node_ip=db_node.ip_address, ks_cf=ks_cf)
+
+        try:
+            with self.cql_connection_patient(db_node) as session:
+                result = session.execute(SimpleStatement('select * from {}'.format(ks_cf), fetch_size=page_size,
+                                                         consistency_level=ConsistencyLevel.ONE))
+                while result.has_more_pages:
+                    result.fetch_next_page()
+            result = {'exit_code': 0, 'stdout': 'finished successfully', 'stderr': ''}
```

Comment:
no need to maintain the same structure as it ran in a subprocess, adapt this to the new code

Reply:
Fixed